data/assets/examples/renderable/renderablecartesianaxes/cartesianaxes_parent.asset

```diff
@@ -0,0 +1,29 @@
+-- With Parent
+-- This example creates a SceneGraphNode that displays coordinate axes of the given parent
+-- node, in this case Earth.
+
+local Node = {
+  Identifier = "RenderableCartesianAxes_Example_Parent",
+  Parent = "Earth",
```

Comment:
Do we want to use the identifier imported from the earth asset here?

Reply:
Yeah, that would be better.

Reply:
However, I do find myself wondering what we said about this when we decided to keep the examples to do a minimal amount of things 🤔 

But in general, to me it seems better to write the examples like we would write them if it was any other asset, to avoid discrepancies and risk of teaching "bad" asset writing to users

Reply:
In other cases I did add the `asset.require`.  I think it is more important to have the examples be loadable in all cases